delivery-kube-config-files/kafka.yaml

```diff
@@ -78,9 +78,41 @@ spec:
             port: 8082
       volumes:
        - name: kafka-persistent-storage
-         awsElasticBlockStore:
-           volumeID: vol-0da5c6a7e25e8f799
-           fsType: ext4
+         persistentVolumeClaim:
+           claimName: kafka-pvc
+
+---
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
It seems we'll need the labels on where the persistent volume relies:   `labels:
    failure-domain.beta.kubernetes.io/region: eu-west-1
    failure-domain.beta.kubernetes.io/zone: eu-west-1a`